<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_fetchpdb</class>
 <widget class="QDialog" name="dlg_fetchpdb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fetch PDBID</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QLabel" name="PdbIDLabel">
       <property name="text">
        <string>Please enter PDBID:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pPdbID">
       <property name="inputMask">
        <string>&gt;NNNN; </string>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pBiologicalUnitCheckBox">
     <property name="text">
      <string>Download Biological Unit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="BioUnitGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Biological Unit:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="BiologicalUnitLabel">
        <property name="text">
         <string>Select Unit Number:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_pBiologicalUnitNumber">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pOk">
       <property name="text">
        <string>Fetch PDB</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_pPdbID</tabstop>
  <tabstop>m_pBiologicalUnitCheckBox</tabstop>
  <tabstop>m_pOk</tabstop>
  <tabstop>m_pCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pOk</sender>
   <signal>clicked()</signal>
   <receiver>dlg_fetchpdb</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlg_fetchpdb</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pBiologicalUnitCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BioUnitGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
